build.rs

```diff
@@ -7,6 +7,10 @@ fn main() {
     let mut cmake = cmake::Config::new("third-party/libswiftnav/");
     let out_dir = env::var("OUT_DIR").unwrap();
 
+    if !cfg!(feature = "testcpp") {
+        cmake.define("libswiftnav_ENABLE_TESTS", "OFF");
+    }
+
     cmake
         .profile("Release")
```

Comment:
Is there a reason we are always using Release profile?  If removed, the cmake-rs crate will dynamically set this based on the Rust release profile.

Reply:
No specific reason as far as I remember, I think we can remove this if the cmake crate selects this automatically